.github/workflows/packagelist-gate.csv

```diff
@@ -1,4 +1,5 @@
 fdk-aac-free,,
+libdb,,
```

Comment:
Cannot be greater than 5.3.28 or whatever you can do to restrict it from the upgrade